### ISSUE TITLE
Can't View Map on Mobile Devices

### ISSUE THREAD

**Issue by GreenflyM** [AHAAAAAAA/PokemonGo-Map#2121]
I'm running the master version on Ubuntu 16.04.1 x64 on a digital ocean droplet. Everything works perfectly on desktop, but I'm unable to load the map when accessing the site from ios or android. Is this a known issue or do I have a configuration error? 

You can view the map here: http://0pokescript.findpoke.net:5000/

I'm entering the following in the console:

python runserver.py -u Username -p password -st 13 -k googleapikeyhere -l "53.32686446 -6.24349594" -H 104.131.7.30

**Comment by rneu31** [comment]
+1

**Comment by rneu31** [comment]
I clicked "options" and it eventually loaded. Not sure if it was just taking a while to load images or whatever.

**Comment by aanc** [comment]
Map load without problem on my Nexus 5 and on my wife's iPhone 6.
Deployed on my home server in Docker.

Could it be related to a connectivity issue of your device ?

**Comment by bdeschut** [comment]
Works fine on my samsung s5

**Comment by domingguss** [comment]
the `Options` side menu does open, but doesn't close anymore (in an iOS app inside a UIWebView (Safari))

**Comment by GreenflyM** [comment]
@aanc Any thoughts on how I could troubleshoot the iphone?

**Comment by Lops88** [comment]
I have the same problem. Tried to run the server from my Ubuntu 15.10 VM. Everything looked fine. Could access the site via wifi from my phone using a dyndns-service. However I was not able to connect when I was using my mobile internet. Running the server now from my Host Win10 system and everything is working. Performance is quite a bit worse on Win10 compared to an Ubuntu VM.

**Comment by aanc** [comment]
@GreenflyM, is your server currently running ? I cannot load http://0pokescript.findpoke.net:5000/ on the desktop, i get connection refused.

Can you specify if it is only the map you can't see (the Option bar displays correctly, but the map stay blank), or if you can't connect at all (total blank page, or error message) ?

**Comment by GreenflyM** [comment]
@aanc Thanks for getting back to me. The server was offline, but is live again if you want to take a look. 

You are correct in understanding that the option bar displays correctly, but the map stays blank.

**Comment by aanc** [comment]
@GreenflyM, I can load the map using your server on my Nexus 5. I'll give it a try tonight with my wife's iPhone (if i don't forget).

Can you load http://maps.google.com successfully on your phone ?

**Comment by GreenflyM** [comment]
Yeah google maps loads fine, the problem seems to be with the app itself. I can load the map on desktop in firefox and chrome, but not safari. On ios it won't load in safari, chrome or opera.

**Comment by techmodo** [comment]
I was able to replicate this issue. The problem only occurs when you are in Incognito or Private mode.

**Comment by invisiblek** [comment]
that patch should allow the javascript to continue if localStorage isn't supported in your browser (which apparently is the case with safari + incognito?)

please test
